### ISSUE TITLE
Practical value of certification

### ISSUE THREAD

**Issue by vchaplygim** [PaulJerimy/SecCertRoadmapHTML#137]
Hi, there is a suggestion to supplement the brief description of all certifications with the following information: 
1. Practical Exam (or Performance based exam) 
2. Theoretical exam (or multi-choice exam) 

It is also possible to visually change the brightness of the color of the block for such a division: the practical exam is a brighter block, the theoretical exam is less bright. 

PS Now it is most often possible to understand theoretical certification or practical certification only by going to the page of the course/certification vendor.

PPS Perhaps it is worth implementing a filter by conditional value among the community separately.
 
 Example, below are valuable certifications, the exam of which is not just answers to questions, but consists in solving practical problems close to production on laboratory stands.

<img width="158" alt="Screenshot 2024-10-01 at 6 33 49 PM" src="https://github.com/user-attachments/assets/53b5c3e3-8e8f-45a2-8803-8904618fce02">
<img width="291" alt="Screenshot 2024-10-01 at 6 33 59 PM" src="https://github.com/user-attachments/assets/6e4d688e-4736-4cda-89ea-27881d44b972">

**Comment by sinecurelife** [comment]
Hi @vchaplygim , in the Javascript version I'm working on I plan to have a lot more information on the certification card. I will definitely add a true/false field for Practical Exam, Theoretical Exam, and interview requirements.

Here's a mock-up of the new certification card:
![image](https://github.com/user-attachments/assets/c7057fa1-7abc-4e7e-b453-900a79f388c6)

**Comment by vchaplygim** [comment]
Hi @sinecurelife, thank you for your answer! It looks great, but I understand that the example does not yet have fields about the practical exam or not?

**Comment by sinecurelife** [comment]
Yes you're right, that option is not on the mock up but is on my feature list. I don't foresee why that data point can't make it to the first version.
